frontend/src/router.ts

```diff
@@ -37,6 +37,15 @@ export const router = new Router({
       path: "/sound-editing/:taskId/quality-check/feedback",
       component: () => import("@/views/QCSubmissionForm.vue")
     },
+    {
+      path: "/listen/:fileName",
+      component: () => import("@/views/Listen/ListenLayout.vue"),
```

Comment:
Why not `AnonymousLayout`? I was expecting you to use it.

Reply:
I used AnonymousLayout but for the particular listening page, I changed it. But I will use AnonymousLayout.